feature/bgp/policybase/ate_tests/route_installation_test/route_installation_test.go

```diff
@@ -181,13 +184,21 @@ func bgpCreateNbr(localAs, peerAs uint32, policy string) *oc.NetworkInstance_Pro
 			nv4.PeerGroup = ygot.String(peerGrpName)
 			nv4.PeerAs = ygot.Uint32(nbr.as)
 			nv4.Enabled = ygot.Bool(true)
-			nv4.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).Enabled = ygot.Bool(true)
+			af4 := nv4.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST)
+			af4.Enabled = ygot.Bool(true)
+			if *deviations.RoutePolicyUnderPeerGroup {
```

Comment:
The deviation name says "policy under peer group", but the actually code says "policy under address family"?

Reply:
HI @xw-g, thanks for the review. The above code protected under deviation is a workaround for not supporting route-policy under peer-group. The this condition is paired with a previous condition in code:
`
if policy != "" && !*deviations.RoutePolicyUnderPeerGroup {
		pg.GetOrCreateApplyPolicy().ImportPolicy = []string{policy}
	}
`
If we are deviated, then we skip the route-policy under peer-group but apply under address family.